polkadot-parachains/launch-runtime/src/lib.rs

```diff
@@ -201,6 +201,10 @@ impl pallet_balances::Config for Runtime {
 	type WeightInfo = weights::pallet_balances::WeightInfo<Runtime>;
 	type MaxReserves = MaxReserves;
 	type ReserveIdentifier = [u8; 8];
+	type HoldIdentifier = ();
+	type FreezeIdentifier = ();
+	type MaxHolds = ();
+	type MaxFreezes = ();
```

Comment:
;)

Reply:
Thanks, addressed in: [d269c3a](https://github.com/encointer/encointer-parachain/pull/186/commits/d269c3a46a8e95037663fda364fe225d7b3f6a10)